UI/DesignWnd.cpp

```diff
@@ -2736,7 +2736,7 @@ void BasesListBox::ResetEmptyListPrompt()
 
 void CompletedDesignsListBox::ResetEmptyListPrompt() {
     if (!GetOptionsDB().Get<bool>("resource.shipdesign.saved.enabled")
-        && !GetOptionsDB().Get<bool>("resource.shipdesign.default.enabled"))
+        && false /*!GetOptionsDB().Get<bool>("resource.shipdesign.default.enabled")*/)
```

Comment:
@geoffthemedio  [In the previous discussion I quoted the the prompt at issue](https://github.com/freeorion/freeorion/pull/2194#discussion_r199333563) and said it looked to me like it was right that the prompt not be shown, and you replied "As you prefer."  Now, I don't understand what your position is.  Are you trying to ask for more time to look into things?

> Haven't thought about it carefully, but

It seems pretty far along in the discussion to be arguing against someone without thinking about it carefully.  Please do think about it carefully.  Please read the prompt and think about the significance of both options, the one that remains and the one that is effectively forced true right now.  

>  If still shown, the prompt might need some rewording, though.

It looks to me like the only situation where even a modified prompt could be appropriate is if someone had modified their content to remove all of the prebuilt designs AND has also set the option to not use any of their own saved designs.  I don't think we need to put time into supporting that use-case-- if they take that step then they should already understand well enough what's going on, and the prompt that does show, ADD_FIRST_DESIGN_DESIGN_QUEUE_PROMPT, should be just fine.

Reply:
I was commenting, not objecting. Do as you prefer, regardless of whether I understand your reasons.

Reply:
> in which I tried to make it as thought he option didn't exist, rather than just that it was always true

and since this goal seems to be a sticking point for you, since you have mentioned it in both discussions now, and it seems to motivate this (to me) strange position you are taking,  I feel compelled to say that goal doesn't much make sense to me.   

The option controlled certain functionality (the loading of prebuilt designs), which we are not taking out nor disabling, without taking out that functionality it is impossible to make it as if the option didn't exist rather than that it is forced to true.   Removing the registration of the option is, by itself, sort of related to making it like it didn't exist, but it is also consistent with it being simply always true.    All of the other changes that you made yourself, except with respect to this one prompt, were equivalent to forcing the option true.

That is the right treatment, treating it like the option is always true, and trying to treat it like it was neither true nor false simply does not make sense in light of the related functionality remaining.

Reply:
It's not a goal or sticking point... It was just how I decided what to do. If you think it should be done differently, that's fine.